scipy/stats/_qmc.py

```diff
@@ -800,11 +837,15 @@ class LatinHypercube(QMCEngine):
 
     """
 
-    def __init__(self, d, centered=False, seed=None):
+    def __init__(
+            self, d: int, centered: bool = False,
+            seed: SeedType = None
+
+    ) -> None:
         super().__init__(d=d, seed=seed)
         self.centered = centered
 
-    def random(self, n=1):
+    def random(self, n: int = 1) -> Union[np.number, np.ndarray]:
```

Comment:
The issue is if someone uses this function and expects a float, he will have a type error in his IDE (because the IDE considers that it should be an `np.ndarray`). Am I correct?

Reply:
But why would someone expect a float? From my understanding, the output is always a `ndarray`. So, it will correctly show a type error, right?

Reply:
Agreed with @tirthasheshpatel, all random are returning `np.ndarray`. The original types from `_qmc.pyi` should be correct and not require much adjustment as we already iterated over this.

Reply:
I return to the previous state in 7a599c601292307a8f438b810b837af976369122 and add a ignore type at the return value